server/test/utils/setup.js

```diff
@@ -0,0 +1,59 @@
+process.env.NODE_ENV = 'test';
+
+const chai = require('chai');
+const path = require('path');
+const chaiHttp = require('chai-http');
+
+const {validCredentials} = require('../mocks/user')
```

Comment:
I noticed that we aren't using a beautifier (like prettier.io) or a linter specifications - we should probably add one for the server and client